tools/roxvet/analyzers/validateimports/analyzer.go

```diff
@@ -241,6 +243,16 @@ func verifyImportsFromAllowedPackagesOnly(pass *analysis.Pass, imports []*ast.Im
 			"pkg/uuid",
 			"pkg/version",
 		)
+
+		// Migrations shall not depend on current schemas. Each migration can include a copy of the schema before and
+		// after a specific migration.
+		if validImportRoot == "migrator" {
```

Comment:
This exception is there for the migrator to:
- read schema version from an existing postgres database
- apply the schema for initial deployment.

Migrations themselves MUST depend on frozen schemas (possibly before and after each specific migration).

Reply:
Yes.